maintaining-and-tuning/monitoring.md

```diff
@@ -153,3 +153,25 @@ access to it can read aggregated metrics about your GRR installation. With these
 about the number of workers, flow activity, and service health can be derived. Make sure to limit
 access to the port, for example by employing a firewall. Furthermore, read
 [Prometheus Security](https://prometheus.io/docs/operating/security/).
+
+## Example visualization setup
+This example will walk you through setting up Grafana as a dedicated visualization software to parse, display, query and set up alerts for the metrics scraped by Prometheus. These instructions assume that GRR server and Prometheus are both up and running.
+
+1. [Install Grafana](https://grafana.com/docs/grafana/latest/installation/#install-grafana) as suggested to your operating system.
+
+2. After Grafana is installed, you may [start the Grafana server](https://grafana.com/docs/grafana/latest/installation/debian/#2-start-the-server) by executing in a terminal (assuming your operating system either Debian or Ubuntu and you installed the latest OSS release):
+```
+sudo systemctl daemon-reload
+sudo systemctl start grafana-server
+sudo systemctl status grafana-server
+```
+This will get the Grafana server up and running on `http://<host>:3000` (usually, `<host>=localhost`).
```

Comment:
I'd expect Grafana to listen on localhost by default. I think it's safe to simply use "localhost" here and elsewhere in the doc.